### DISCUSSION TITLE
Back to School: Summer Experience Roundup

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#66911] [GitHub Education]
![Experience Logo Multiple](https://github.com/community/community/assets/107881423/7f940465-256a-4bf1-8895-2742352a6947)

In case you missed it, we launched a few Experiences over the summer. Collaborating with our Student Developer Pack partners including Scrimba, New Relic, Dev Cycle, Bump.sh, LambdaTest and more, check out these Experiences you may have missed.

## Launchpad: Intro to Javascript 

What is a GitHub Global Campus Launchpad?
As a part of the Global Campus suite of Experiences, Launchpads function as a sub-genre providing small bites of content designed for learners to gain a foundational CS skill in under twenty minutes. 

Build confidence in your coding skills right away with this Launchpad introduction to Javascript by using Student Developer Pack Partner [Scrimba](https://scrimba.com/)’s interactive interface that allows you to code within the tutorial itself. With this Experience, students learn to write their first lines of code in just 15 minutes with a scrim to guide them through.

## Mobile App Development

The Mobile App Development Experience gives you the tools, resources and inspiration to design, build, test and deploy your next mobile application.
- Kick off your project with a mobile template in Codespaces or browse through the courses on offer from FrontEndMasters where you can learn how to build responsive applications using Flutter, Swift, Kotlin and more.
- Explore pack offers from Bump, Replit, Kodika and Azure to build, document and deploy your code. Level up by making sure your app looks and functions as it should, and is as accessible as possible, with the help of LambdaTest and LingoHub.
- Don't underestimate the power of open source when building your mobile app. Explore free resources from NativeScript and Flutter to make your mobile apps a reality.

## DevOps
Developer Operations (DevOps) is the symbiotic relationship between teams, culture and development that allows for collaboration, growth and continuous improvement. Used as a set of practices and tools, DevOps integrates and automates the work of software development and operations as a means for improving the development life cycle.

Redeem offers from Sentry, TravisCI, Browserstack, DevCycle, Codescene, New Relic as you explore what a DevOps Engineer does and what kinds of skills it takes to become one.

We’re thrilled to provide Global Campus as not only a peer led space to explore, collaborate and contribute to projects, but as a place to experiment with industry leading tools that can expand any skillset through Experiences. To leverage offers from these partners and all of our [Student Developer Pack](https://education.github.com/pack?utm_source=github-blog-pack-partner-highlights) partners is to first become a verified student in Global Campus. 

What is a Primer exactly? What about a Launchpad?[ View this guide to learn](https://github.com/education/Global-Campus-Experiences) how to tell the difference between different types of Experience categories and what they mean for your learning journey.

**Comment by AvinashShrivastav** [comment]
Seems quite interesting!

**Comment by baimamboukar** [comment]
This is interresting ✨🚀

**Comment by Natheerasm** [comment]
this is intresting

**Comment by farazsfa007** [comment]
It appears rather intriguing👌😎

**Comment by bat0ri** [comment]
))
